template_config.yml

```diff
@@ -29,6 +29,7 @@ core_import_allowed:
 - pulpcore.app.*viewsets
 - pulpcore\.app.*admin
 - ProgressReportSerializer
+- pulpcore.app.*tasks
```

Comment:
This change is needed until we have the issue https://pulp.plan.io/issues/9418 is resolved